index.html

```diff
@@ -83,121 +83,97 @@ <h2 id="recent-2" class="font-family headline-paragraphs p3">My Recent Works</h2
-              <p class="techs-items-basic techs-items-1">Ruby on rails</p>
-              <p class="techs-items-basic techs-items-2">css</p>
-              <p class="techs-items-basic techs-items-3">JavaScript</p>
-              <p class="techs-items-basic techs-items-4">html</p>
             </div>
             <div class="flex-item-1 flex-container flex-column-center-axis">
-              <a href="#" class="bttn font-family">See Project</a>
+              <a href="#" class="bttn font-family popup" id="3">See Project</a>
             </div>
           </div>
         </div>
       </div>
       <div class="grid-container flex-item-projects">
-        <div class="grid-item-1">
+        <div class="grid-item-1 project-image-container">
         </div>
         <div class="grid-item-2">
           <div class="flex-container flex-column-center-axis works-title font-family">
             <div class="flex-item-1 flex-container flex-column-center-axis">
               <h3 class="card-font card-Title font-family">Multi-Post Stories<br><span>Gain+Glory</span></h3>
             </div>
             <div class="flex-item-2 flex-container techs-items-font font-family">
-              <p class="techs-items-basic techs-items-1">Ruby on rails</p>
-              <p class="techs-items-basic techs-items-2">css</p>
-              <p class="techs-items-basic techs-items-3">JavaScript</p>
-              <p class="techs-items-basic techs-items-4">html</p>
             </div>
             <div class="flex-item-1 flex-container flex-column-center-axis">
-              <a href="#" class="bttn font-family">See Project</a>
+              <a href="#" class="bttn font-family popup" id="4">See Project</a>
             </div>
           </div>
         </div>
       </div>
       <div class="grid-container flex-item-projects">
-        <div class="grid-item-1">
+        <div class="grid-item-1 project-image-container">
         </div>
         <div class="grid-item-2">
           <div class="flex-container flex-column-center-axis works-title font-family">
             <div class="flex-item-1 flex-container flex-column-center-axis">
               <h3 class="card-font card-Title font-family">Multi-Post Stories<br><span>Gain+Glory</span></h3>
             </div>
             <div class="flex-item-2 flex-container techs-items-font font-family">
-              <p class="techs-items-basic techs-items-1">Ruby on rails</p>
-              <p class="techs-items-basic techs-items-2">css</p>
-              <p class="techs-items-basic techs-items-3">JavaScript</p>
-              <p class="techs-items-basic techs-items-4">html</p>
             </div>
             <div class="flex-item-1 flex-container flex-column-center-axis">
-              <a href="#" class="bttn font-family">See Project</a>
+              <a href="#" class="bttn font-family popup" id="5">See Project</a>
             </div>
           </div>
         </div>
```

Comment:
- [ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript. Kindly work on this